src/workerd/api/performance.c++

```diff
@@ -188,7 +192,8 @@ jsg::Ref<PerformanceMeasure> Performance::measure(jsg::Lock& js,
     kj::String measureName,
     kj::OneOf<PerformanceMeasure::Options, kj::String> measureOptionsOrStartMark,
     jsg::Optional<kj::String> maybeEndMark) {
-  double startTime = now();
+  Worker::Isolate::from(js).getLimitEnforcer().markPerfEvent("performance_measure"_kjc);
```

Comment:
nit: we can do this later, but `performance.measure()` will typically take the name of two previous `performance.mark()` calls and be the duration of time between them. If we're capturing the individual marks, then it might be worthwhile to pass the start and end marks for the measure through the event.